### ISSUE TITLE
storage error messages from mkfs.xfs are too verbose

### ISSUE THREAD

**Issue by allisonkarlitskaya** [cockpit-project/cockpit#17672]
### Explain what happens

![image](https://user-images.githubusercontent.com/36541154/186694618-bf4081df-1849-424f-b3b5-bd929f64e676.png)

**Comment by mvollmer** [comment]
Yeah.  The strategy so far is to avoid triggering errors.  For example, we have code to prevent passing too long labels when formatting XFS: https://github.com/cockpit-project/cockpit/blob/main/pkg/storaged/utils.js#L144

In addition, we can trim everything after "Usage: ". *shrug*

**Comment by jelly** [comment]
![image](https://user-images.githubusercontent.com/67428/186855576-d058509c-3bb4-48a3-85c4-609b3f9c0f15.png)

**Comment by garrett** [comment]
The bold part of the alert is the title. The contents shouldn't be the title.

https://www.patternfly.org/v4/components/alert#inline-variations

![image](https://user-images.githubusercontent.com/10246/188599432-e3c365a9-36d4-4306-a2d9-76373008aee9.png)

We have been using our own custom solution for expandable errors (I think with the view details link that expands?), but PatternFly "recently" added expandable alerts:

https://www.patternfly.org/v4/components/alert#expandable

![image](https://user-images.githubusercontent.com/10246/188601608-301cd929-e528-4ad5-9548-b21c98500c86.png)

![image](https://user-images.githubusercontent.com/10246/188601652-2de87f4d-bab0-4f58-b083-80ef8ea8edd9.png)

These are all "success" alerts in the examples. We'd use errors/warnings. (Alerting on success isn't generally a good pattern, but PF uses them in the examples.)
